lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -371,7 +371,7 @@ sub engine_workit {
 
     # os-autoinst's commands server
     $job_info->{URL}
-      = "http://localhost:" . ($job_info->{settings}->{QEMUPORT} + 1) . "/" . $job_info->{settings}->{JOBTOKEN};
+      = "http://localhost:" . ($job_settings->{QEMUPORT} + 1) . "/" . $job_settings->{JOBTOKEN};
```

Comment:
Technically there's unnecessary double quoting here too.

Reply:
ok, I have a draft PR with further refactoring in https://github.com/os-autoinst/openQA/pull/3807 where I will include your proposal

Reply:
done